modules/services/model/src/main/java/au/com/shiftyjelly/pocketcasts/models/entity/Podcast.kt

```diff
@@ -59,6 +60,7 @@ data class Podcast(
     @ColumnInfo(name = "show_notifications") var isShowNotifications: Boolean = false,
     @ColumnInfo(name = "show_notifications_modified") var showNotificationsModified: Date? = null,
     @ColumnInfo(name = "auto_download_status") var autoDownloadStatus: Int = 0,
+    @ColumnInfo(name = "auto_download_limit") var autoDownloadLimit: AutoDownloadLimitSetting = AutoDownloadLimitSetting.TWO_LATEST_EPISODE,
```

Comment:
This default value leads to some significant issues. When I sign in and to a clean sync all of the podcasts have `auto_download_limit` set to `TWO_LATEST_EPISODES`. Moreover, all podcasts that I see in discover have this column set to `TWO_LATEST_EPISODES`.

Reply:
I am going to update it to default OFF and change to TWO_LATEST_EPISODE after the user subscribes to the podcast